tenseal/proto/tensors.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+import "tensealcontext.proto";
+package tenseal;
+
+//BFVVector parameters
+message BFVVectorProto {
+    // The context protobuffer
+    TenSEALContextProto context = 1;
+    // The size of the encrypted vector
+    uint32 size = 2;
+    // The serialized ciphertext
+    bytes ciphertext = 3;
+};
+
+//CKKSVector parameters
+message CKKSVectorProto {
+    // The context protobuffer
+    TenSEALContextProto context = 1;
```

Comment:
You are saving the context into the vectors, which can be avoided and can reduce a lot of space. An example use case is where someone wants to send a bunch of encrypted vectors for outsourced computation, so he will only need to send a single context, and multiple encrypted vectors, that when received should be linked to the received context.

Can we have a ts.ckks_vector_from(buffer, context) kind of API which links the vector to the specified context without the need to include it in the vectors' protobuff?